tdrs-backend/tdpservice/parsers/schema_defs/tanf/t3.py

```diff
@@ -66,50 +158,141 @@
     preparsing_validators=[
         validators.notEmpty(start=60, end=101),
     ],
-    postparsing_validators=[],
+    postparsing_validators=[
+        validators.if_then_validator(
```

Comment:
im still having trouble understanding why only the 2nd child record with invalid SSN is being caught @elipe17 @raftmsohani 
![2369p5](https://github.com/raft-tech/TANF-app/assets/63075587/e5bf99f5-748c-4180-aa95-e46c85c24ff0)
![2369p6](https://github.com/raft-tech/TANF-app/assets/63075587/dbedcd7f-fd9a-4124-8381-7d06eb24b51d)